ui/webui/test/helpers/step_logger.py

```diff
@@ -23,6 +24,8 @@ def log_step(snapshots=False, snapshot_before=False, snapshot_after=False, docst
     :type docstring: bool, optional
     """
     def decorator(function):
+        if not os.environ.get('PERMIAN', False):
```

Comment:
The ENV variable name should be different, it has nothing to do with Permian. And I think it should only have an effect on the creation of the screenshots. Logging what steps are being performed is always useful and AFAIK has no negative effects.

Reply:
Yes & more detailed logs are especially useful when we hit race conditions.